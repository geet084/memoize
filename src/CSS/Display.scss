.display {
  display: flex;
  justify-content: center;
}

.intro {
  color: #fff;
  width: 450px;
  text-align: center;
  background-color: #2D4C46;
  padding: 40px;
  opacity: .8;
  font-size: 3rem;
  border-radius: 15px;
}