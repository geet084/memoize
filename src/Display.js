import React, { Component } from 'react';
import './CSS/Main.scss';
import Card from './Card.js'

export default class Display extends Component {
  constructor() {
    super();

    this.state = {
      correctlyAnswered: null
    }
  }

  render() {
    return (
      <section>
        <Card questions={this.props.prototypes}
          answer={this.props.answer}
          answered={this.props.answered} />
      </section>
    )
  }
}