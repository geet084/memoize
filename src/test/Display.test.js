import React from 'react';
import Display from '../Display.js';
import { shallow } from 'enzyme';

describe('Display', () => {
  let wrapper;

  // beforeEach(() => {
  //   wrapper = shallow(
  //     <Display />
  //   )
  // })

  it('should match the snapshot with all data ', () => {
    expect(wrapper).toMatchSnapshot();
  })

});

