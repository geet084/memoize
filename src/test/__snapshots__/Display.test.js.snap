// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display should match the snapshot with all data  1`] = `undefined`;
